hypervisor/src/mshv/mod.rs

```diff
@@ -310,6 +310,13 @@ impl hypervisor::Hypervisor for MshvHypervisor {
         )
         .map_err(|e| hypervisor::HypervisorError::SetPartitionProperty(e.into()))?;
 
+        // Always create a frozen partition
+        fd.set_partition_property(
```

Comment:
can you call `fd.freeze_vm()` implemented below?

Reply:
Not really. We do not implement freeze/oase for fd(from mshv crate)

Reply:
My mistake, i meant `vm.freeze_vm()` (or now, `vm.pause()`). But, I realize the vm object is not constructed yet in this function, so you can ignore